common/environments/python-sdk-v2/environment.yaml

```
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/python-sdk-v2
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
+    - conda_dependencies.yaml
+  publish:
+    location: mcr
+    visibility: public
```

To match your mcr PR:
```suggestion
    visibility: unlisted
```